docs/examples/extension.ipynb

```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Typo in "continuously". IIRC, having <code>central_condition_inherent=True</code>  actually means the satellite component is <em>not</em> multiplied by the central component, since it is assumed that the occupation is zero inherently when the satellite occupation is (and thus doesn't have to be manually forced to be).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/halomod/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>